src/screens/onboarding/container/index.tsx

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
```

Comment:
Eu curto usar um plugin do eslint pra organizar os imports, se vc achar a idéia interessante dá uma olhada nesse post aqui com um tutorial de como fazer: https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m?utm_source=pocket_mylist

Reply:
Opa blz man valeu!! vou olhar!! No projeto ja configurei o eslint para trabalhar em conjunto com o prettier. Acho que para isso basta adicionar uma nova regra no eslintrc.json